src/flow/src/batching_mode/engine.rs

```diff
@@ -185,9 +187,9 @@ impl BatchingEngine {
         let flow_type = flow_options.get(FlowType::FLOW_TYPE_KEY);
 
         ensure!(
-            flow_type == Some(&FlowType::Batching.to_string()) || flow_type.is_none(),
+            matches!(flow_type, Some(ty) if ty == "batching"),
```

Comment:
Don't forget the `None`.

Reply:
Does it need to check if it is `None`? It was checked before refactoring.

Reply:
> Does it need to check if it is `None`? It was checked before refactoring.

Yes, as default behavior is to use batching mode if no type was given?